web/rest/platform/config/api/raw/provider.yml

```diff
@@ -149,8 +149,8 @@ Ivoz\Provider\Domain\Model\Carrier\Carrier:
     access_control: '"ROLE_SUPER_ADMIN" in roles && user.hasAccessPrivileges(_api_resource_class, request.getMethod())'
 
 Ivoz\Provider\Domain\Model\Company\Company:
-  itemOperations: []
-  collectionOperations: []
+  collectionOperations:
```

Comment:
My bad, `itemOperations: []` was actually required. 
Add tests to assert that PUT, DELETE and GET with id return 405 status code